packages/tools/tests/test/playwright/visualizationPlaywright.utils.ts

```diff
@@ -37,20 +37,16 @@ export const evaluatePlaywrightVisTests = async (engineType = "webgl2", testFile
         return !(externallyExcluded || test.excludeFromAutomaticTesting || (test.excludedEngines && test.excludedEngines.includes(engineType)));
     });
 
-    function log(msg: any) {
+    function log(msg: any, title?: string) {
         if (logToConsole) {
-            console.log(msg);
+            console.log(title? `[${title}]` : '', msg);
         }
         if (logToFile) {
             fs.appendFileSync(logPath, msg + "\r\n", "utf8");

```

Comment:
Shouldn't the log to file be updated as well?

Reply:
Done